R/mergeSEs.R

```diff
@@ -807,11 +807,11 @@ setMethod("mergeSEs", signature = c(x = "list"),
 ########################### ..get_unique_names ###########################
 # This function convert colnames unique
 
-# Input: TreeSEs and MARGIN
+# Input: TreeSEs and by
 # Output: One TreeSE with unique sample names compared to other TreeSE
-.get_unique_names <- function(tse1, tse2, MARGIN, suffix=2){
-    # Based on MARGIN, get right names
-    if( MARGIN == "row" ){
+.get_unique_names <- function(tse1, tse2, by, suffix=2){
+    # Based on by, get right names
+    if( by == "row" ){
         names1 <- rownames(tse1)
```

Comment:
Revert changes in this internal function. These are not needed in order to get the new parameter name work.

Reply:
Do as little changes as possible as this is already working code, It makes the review also harder if there are extra changes

Reply:
Sorry for the frustration I may have caused. I was going about it differently.